PR CL-24-Add-Authantification

OleksandraKondratenko opened the pull request.

sinerger committed:
Create Core project

sinerger committed:
Merge pull request #56 from kovalenko-Max/CL-90-create-project-crmlitecore-and-add

PR CL-90 Create project CRMLite.Core and add policy, roles

MykhailoKnysh commented on CRMLite.CRMAPI/Controllers/LeadController.cs:
```diff
@@ -14,13 +15,16 @@ public class LeadController : ControllerBase
 
-        public LeadController(ILeadService leadService, IConfirmMessageService confirmMessageService, IBusControl busControl)
+        public LeadController(ILeadService leadService, IConfirmMessageService confirmMessageService
+            , IBusControl busControl, ISessionService jWTTokenHelper)
```
Move comma to previous line

Aryewar committed:
Add Authantification

Add Project and add CRUD


Added LeadRepositories

CreateService

Fix Naming CRMMicroservice


Added Autofac


Add InsihghtDB


Add InsihghtDB

PR CL-23-Add-Registration

PR CL-23-Add-Registration

Configured Serilog

Add SMTP server


PR CL-24-Add-Authantification

fixup! PR CL-24-Add-Authantification

fixup! fixup! PR CL-24-Add-Authantification
